src/main/java/net/sf/jabref/gui/groups/UndoableMoveGroup.java

```diff
@@ -39,8 +38,7 @@
 
 
     public UndoableMoveGroup(GroupSelector groupSelector, MoveGroupChange moveChange) {
-        this.groupSelector = Objects.requireNonNull(groupSelector);
-
+        Objects.requireNonNull(groupSelector);
         Objects.requireNonNull(moveChange);
```

Comment:
In that case, I would move the requireNonNull check to the first usage as a the requireNonNull also returns the passed object.